sevenzipjbindings/src/main/java/com/mucommander/sevenzipjbindings/SevenZipJBindingROArchiveFile.java

```diff
@@ -79,8 +77,20 @@ public SevenZipJBindingROArchiveFile check() throws IOException {
 
     private IInArchive openInArchive() throws IOException {
         if (inArchive == null) {
-            SignatureCheckedRandomAccessFile in = new SignatureCheckedRandomAccessFile(file, formatSignature);
-            inArchive = SevenZip.openInArchive(sevenZipJBindingFormat, in, password);
+            Pattern multiPartRarPattern = Pattern.compile("[.]part\\d+[.]rar");
+            Matcher matcher = multiPartRarPattern.matcher(file.getAbsolutePath());
```

Comment:
we can match against the filename (rather than the whole path), right?

Reply:
sure